src/js/taskMenager.js

```diff
@@ -110,6 +110,13 @@ export default class TaskMenager {
 
     this.setStoredState();
   };
+
+  searchTasks = (searchText) => {
+    this.getStoredState();
+    let { todoTasks, doneTasks } = this;
+    this.todoTasks = todoTasks.filter(({ name }) => name.toLowerCase().includes(searchText.toLowerCase()));
+    this.doneTasks = doneTasks.filter(({ name }) => name.toLowerCase().includes(searchText.toLowerCase()));
```

Comment:
Don't mutate taskMenagerState, this method should return object with matched todo and done tasks